src/pages/ViewEmail/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+function ViewEmail() { 
```

Comment:
At this point it's worth renaming this component to be consistent with the other pages - just put Page on the end. Make sure you rename both the folder and the function